src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,6 +1,12 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.IllegalDateParametersException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    public static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
it's still public

Reply:
funny, but its private... 
![image](https://user-images.githubusercontent.com/56148428/88926071-1c41ad80-d27e-11ea-9ab7-b83748ae99c9.png)

Reply:
and now?